serverless.yml

```diff
@@ -36,6 +36,7 @@ package:
 plugins:
   - serverless-offline-python
   - serverless-python-requirements
+  - serverless-dynamodb-local
```

Comment:
The docker version can be accessed by running `make test`. I would say give that a try and if this approach seems better, than keep it. I would recommend updating the README as well, now that you've experienced setting it all up from scratch.

Reply:
Removed it :)